Readme.rst

```diff
@@ -107,12 +55,13 @@ Contributions
 Contributions are always welcome and encouraged!
 Please open any issues in the Project Mu GitHub tracker and read https://microsoft.github.io/mu/How/contributing/
 
+For documentation:
 
 Copyright & License
 ===================
 
-Copyright (c) Microsoft Corporation.
-SPDX-License-Identifier: BSD-2-Clause-Patent
+| Copyright (C) Microsoft Corporation

```

Comment:
Should stay lowercase 'c', right?

Reply:
Fixed in latest push, and fixed in other repos as well.